templates/vue/src/components/NodeModal.vue

```diff
@@ -535,6 +536,12 @@ export default {
         })
       }
       await this.updateLockedStatus()
+      this.updateSelectedNode(this.node.id)
+      this.$router.push({
+          name: names.APP,
+          params: { nodeId: this.node.id },
+          query: this.$route.query,
+        })
```

Comment:
@wynnset That sounds good, I think these lines might be a fix that we want to add regardless (including lines 525-527 and 479)